tests/server/3-document.spec.js

```diff
@@ -208,5 +191,19 @@ describe('Document', () => {
               res.body.message.should.equal('Document not found');
               done();
           });
-    })
-})
+    });
+    
+    it('should return all regular documents if role=regular query is used', (done) => {
+        server
+          .get('/api/documents?role=regular')
+          .set('authorization', userToken)
+          .end((err, res) => {
+              console.log(res.body);
```

Comment:
### Critical

Console logs should not exist in production code, feedback has already been delivered multiple times on this.